data/json/items/generic.json

```diff
@@ -826,6 +826,34 @@
     "weight": "5 g",
     "volume": 0
   },
+  {
+    "type": "GENERIC",
+    "id": "delay_fuze",
+    "symbol": ",",
+    "color": "light_gray",
+    "name": "delayed fuze",
+    "category": "spare_parts",
+    "description": "A complex mechanical fuze.  It seems it can be used to detonate stable explosives after a short time by triggering it",
```

Comment:
Period in the end.

Reply:
Got it, thanks pryanik. will go double check elsewhere for periods at the end of sentence.